### ISSUE TITLE
Base16 Tomorrow Dark - Incorrect 

### ISSUE THREAD

**Issue by scottgreenup** [AndrsDC/base16-vscode#6]
https://github.com/AndrsDC/base16-vscode/commit/3e1e9fc7e2552ae608ab1012a9a2ee8451a88983

It seems that the transition from the old format to the new format for Tomorrow Dark might be incorrect?

For example the foreground color in the old version is 1d1f21 where-as the new color is d6d6d6.

The colors in VSCode are also very wrong.

**Comment by pzinovkin** [comment]
Before:
<img width="654" alt="screen shot 2018-02-01 at 09 47 48" src="https://user-images.githubusercontent.com/440630/35664946-15c51dc8-0735-11e8-9e9f-368dc4ab692b.png">

After:
<img width="652" alt="screen shot 2018-02-01 at 09 38 12" src="https://user-images.githubusercontent.com/440630/35664951-1d9a66de-0735-11e8-89be-c93134ac1407.png">

**Comment by zeezali** [comment]
Yeah I noticed the same thing as well after being updated to the latest version 😕

**Comment by AndrsDC** [comment]
I got the new files from updated .json files made for VSCode, while the old files were converted from .tmTheme. I will look look into it. Maybe I'll revert to the old files but I'll see if there's another solution.

**Comment by jryans** [comment]
I recently encountered this as well. I guess there's no way to revert to old versions of extensions... 😞 

For the moment, I published a fork of this that reverts back to v1.4.0, as I wasn't sure what the correct fix would be to submit a PR here.

https://marketplace.visualstudio.com/items?itemName=jryans.base16-themes

I'll unpublish my fork once some kind of fix is found here.

**Comment by cold-coder** [comment]
Actually, if you guys only need Base 16 Tomorrow Dark theme, there is an solo extension called [Base16 Tomorrow Dark+](https://marketplace.visualstudio.com/items?itemName=Shurelia.base16-tomorrow-dark-vscode), that works fine for me.

**Comment by AndrsDC** [comment]
What I did on the latest update was take the latest _.thTheme_ files for this themes and convert them with the built in tool in VSCode. I'll have to hunt for old-ish files.
